website/templates/emails/mentions.txt.mako

```diff
@@ -0,0 +1,3 @@
+${user} at ${localized_timestamp}: ${content}.
```

Comment:
Is the content and UI for this email in the spec? I would think we'd want to use our existing notifications HTML email template for this.